.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -47,7 +47,7 @@ body:
   attributes:
     label: Windows User Only
     description: Are you using VPN, Proxy, Special Firewall rules, Security Software or custom Activity directory features? if Yes, please describe.
-    placeholder: "e.g. VPN PulseSecure, Kaspersky Total Security 21.2.x, custom proxy configs or activity directory features..."
+    placeholder: "e.g. VPN PulseSecure, Kaspersky Total Security 21.2.x, custom proxy configs, activity directory features or N/A"
```

Nitpicking, but there should be a comma before the last element of the list:

```suggestion
    placeholder: "e.g. VPN PulseSecure, Kaspersky Total Security 21.2.x, custom proxy configs, activity directory features, or N/A"
```

---

Thanks for suggesting Jan, I'll fix that.